desk/app/activity.hoon

```diff
@@ -548,7 +548,8 @@
   =/  collapsed
     (~(gas in collapsed.acc) (turn top head))
   :-  (~(put by sources.acc) source src-info(added &))
-  [(sub limit.acc 1) (snoc happenings.acc [source time top]) collapsed]
+  =/  happening  [source (head (head top)) top]
+  [(sub limit.acc 1) (snoc happenings.acc happening) collapsed]
```

Comment:
> the event on which we built this bundle (...) may not be present in the bundle itself

That sounds wild, not sure I understand why that would be the case.

Either way the `top` here is a `list`, I don't think it's strictly guaranteed to contain at least one item right? So this may runtime crash in those cases. Are we ok with that?

(`+<.top` would probably be more idiomatic, too.)